.github/workflows/main.yml

```diff
@@ -96,11 +96,20 @@ jobs:
           if [[ ${{ matrix.torch-version }} != "1.5.0" && ${{ matrix.python-version }} == "3.9" ]]; then mypy forte; fi
       - name: Test with pytest and run coverage
         run: |
-          coverage run -m pytest tests
+          coverage run -m pytest tests  --ignore=tests/forte/notebook_tutorial
           coverage run --append -m pytest --doctest-modules forte
       - name: Upload coverage
         run: |
           codecov
+      - name: Test notebook tutorials with installing Fortex
+        run: |
+          git clone https://github.com/asyml/forte-wrappers.git
+          cd forte-wrappers
+          pip install src/huggingface
+          pip install src/nltk
+          cd ..
+          coverage run -m pytest tests/forte/notebook_tutorial
```

Comment:
I guess there will be different notebooks depending on different packages. They will conflict eventually. 

One way to do this maybe similar to our setup here: 
1. https://github.com/asyml/forte-wrappers/blob/main/.github/workflows/main.yml#L83
2. https://github.com/asyml/forte-wrappers/blob/main/.github/workflows/main.yml#L119